app/assets/stylesheets/partials/_footer.scss

```diff
@@ -1,35 +1,31 @@
-
+// Footer links list
 .ea-footer {
-  font-size: 14px;
-  font-weight: 400;
-  line-height: 1.14286;
-  padding-left: 15px;
-  padding-right: 15px;
+  padding-right: 15px; // TODO use preset
+  font-size: 14px; // TODO use preset
+  line-height: 1.14286; // TODO use preset
+  font-weight: 400; // TODO use preset
   text-transform: none;
-
-  .ea-footer li {
-    font-size: 14px;
-    font-weight: 400;
-    line-height: 1.14286;
-    margin-right: 10px;
-    text-transform: none;
-  }
-
-  .inline {
-    display: inline;
+}
+@media (min-width: 641px){
+  .ea-footer {
+    padding-right: 30px;
+    font-size: 16px; // TODO use preset
```

Comment:
how about i change it to read :

 // TODO - can we use the preset variables in place of these hard coded pixels ?

else Martin happy if we just delete it